portal/patient_self_checkin/step2.php

```diff
@@ -0,0 +1,111 @@
+<?php
+$ignoreAuth = true;
+require_once "../../interface/globals.php";
+require_once $GLOBALS['srcdir'] . '/sql.inc';
+?>
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+          crossorigin="anonymous">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css"
+          integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp"
+          crossorigin="anonymous">
+
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
+            integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
+            crossorigin="anonymous"></script>
+
+    <style>
+
+        .btn-lg {
+
+            width: 200px;
+            height: 150px;
+            margin: 10px;
+            font-size: 30px;
+        }
+
+        .col-sm-3 {
+
+            text-align: center;
+        }
+
+        h2 {
+
+            text-align: center;
+        }
+
+        body {
+            font-family: sans-serif;
+            background-color: #638fd0;
+
+            background: -webkit-radial-gradient(circle, white, #638fd0);
+            background: -moz-radial-gradient(circle, white, #638fd0);
+        }
+
+    </style>
+
+<?php
+if ($GLOBALS['self_checkin_enable'] == 1) {
+    // Store variable from previous step
+    $sex = htmlspecialchars($_GET['sex']); ?>
+
+    <!-- // Get month of birth -->
+
+    <h2>Select your month of birth</h2>
+
+    <div class="row">
+        <div class="col-sm-3"><a href="step3.php?sex=<?php echo $sex; ?>&month=01" class="btn btn-lg btn-default">
```

Comment:
note it makes more sense to html escape here (and a good place to go over the html escaping functions in openemr):
text() does html escaping outside html attributes
attr() does html escaping inside html attributes.
``````
<div class="col-sm-3"><a href="step3.php?sex=<?php echo attr($sex); ?>&month=01" class="btn btn-lg btn-default">